ws-nextjs-app/pages/[service]/send/[id]/FormField/InvalidMessageBox.tsx

```diff
@@ -46,37 +46,53 @@ const formatValidationMessage = (
   return message;
 };
 
-export default ({
-  id,
-  messageCode,
-  hasArrowStyle = true,
-  suffix,
-  validation,
-}: InvalidMessageBoxProps) => {
-  const {
-    translations: { ugc = fallbackTranslations },
-  } = useContext(ServiceContext);
+const InvalidMessageBox = forwardRef(
+  (
+    {
+      id,
+      messageCode,
+      hasArrowStyle = true,
+      isErrorSummary = false,
+      suffix,
+      validation,
+    }: InvalidMessageBoxProps,
+    ref: ForwardedRef<HTMLElement>,
+  ) => {
+    const {
+      translations: { ugc = fallbackTranslations },
+    } = useContext(ServiceContext);
 
-  const message = formatValidationMessage(
-    ugc[messageCode ?? InvalidMessageCodes.FieldRequired] ?? '',
-    validation,
-  );
+    const message = formatValidationMessage(
+      ugc[messageCode ?? InvalidMessageCodes.FieldRequired] ?? '',
+      validation,
+    );
 
-  return (
-    <>
-      {hasArrowStyle && <div css={styles.errorArrow} />}
-      <div css={styles.errorMessageBox(hasArrowStyle)}>
-        <ErrorSymbol />
-        <Paragraph
-          id={id}
-          css={styles.errorText}
-          fontVariant="sansBold"
-          size="minion"
+    return (
+      <>
+        {hasArrowStyle && <div css={styles.errorArrow} />}
+        <div
+          css={[styles.errorMessageBox, !hasArrowStyle && styles.hasArrowStyle]}
         >
-          {message}
-          <VisuallyHiddenText>{`, ${suffix}`}</VisuallyHiddenText>
-        </Paragraph>
-      </div>
-    </>
-  );
-};
+          <ErrorSymbol />
+          <Text
+            id={id}
+            css={[
+              styles.errorText,
+              isErrorSummary && styles.focusIndicatorErrorSummary,
+            ]}
+            fontVariant="sansBold"
+            size="minion"
+            as="strong"
+            {...(isErrorSummary && { tabIndex: -1 })}
```

Comment:
Please Could we add a note to the PR to say that we tested moving focus to the wrappig div and decided to place focus here as it was announced better etc